bin/sdl_run

```diff
@@ -51,37 +51,41 @@ if [ -z ${SAMPLE} ]; then usage; fi
 if [ -z ${NEVENTS} ]; then NEVENTS=-1; fi
 if [ -z ${TAG} ]; then usage; fi
 
-# If it will not be compiled, then make sure that it was precompiled
-if ${PRECOMPILED}; then
-  if [ ! -f ${TRACKLOOPERDIR}/bin/sdl ] || ( [ ! -f ${TRACKLOOPERDIR}/SDL/libsdl_cuda.so ] && [ ! -f ${TRACKLOOPERDIR}/SDL/libsdl_cpu.so ] ); then
-    echo "SDL library or binary has not been compiled. Please use -f flag."
-    usage
-  fi
-fi
-
 # If a backend is specified then make sure that corresponding library exists
 # and set the environment variable to preload it.
 if [ -z ${BACKEND} ]; then
   BACKEND="default"
```

Comment:
If you use `sdl_run` without `-b` then `BACKEND_INT` is not set, so the `sdl` command fails.

Reply:
I defaulted to `cuda`  to preserve the historical choice.

There is a formal conflict though, the `bin/sdl` in multi-arch build defaults to `cpu` to get the CI going.

Reply:
Okay, it makes sense to default to cuda. I'll adjust the CI to take this into account.